praktikumsplaner-frontend/frontend/src/components/praktikumsplaetze/Meldung/AusbildungsJahrSelect.vue

```diff
@@ -49,13 +49,24 @@ import Praktikumsstelle from "@/types/Praktikumsstelle";
 
 const validationRules = useRules();
 
-const properties = defineProps<{
+interface Properties {
```

Comment:
This is because of 'props'-Part in the scoped slot in line 32. There it should be "props" https://vuetifyjs.com/en/api/v-select/#slots-item
So if I give both the same name, it unfortunately causes problems. So unfortunately I have to name them differently. It bothers me too :(